generate-new/src/Bespoke.hs

```diff
@@ -756,14 +756,14 @@ boolConversion = genRe "Bool conversion" $ do
   tellExport (ETerm (TermName "bool32ToBool"))
   tellImport 'bool
   tellDoc [qqi|
-    boolToBool32 :: Bool -> {bool32}
-    boolToBool32 = bool {false} {true}
+boolToBool32 :: Bool -> {bool32}
+boolToBool32 = bool {false} {true}
```

Comment:
lol, what happened to all my quasiquotes

Reply:
On my end, the spaces at the start of the lines in the quasi quotes were being included in the generated source, which caused parse errors. No idea why the behaviour is different on my end than yours... obviously feel free to ditch this if this causes issues on your end now.

Reply:
weird!

Reply:
Perhaps this doesn't work on windows? https://github.com/expipiplus1/vulkan/blob/43222eecf54c41546bbcda1e4b467c7ba35d18a6/generate-new/src/Text/InterpolatedString/Perl6/Unindented.hs